### ISSUE TITLE
NPE in ForwardExtentCopy

### ISSUE THREAD

**Issue by mibby** [IntellectualSites/FastAsyncWorldEdit-1.13#82]
Compiled FAWE 1.13 as of commit https://github.com/IntellectualSites/FastAsyncWorldEdit-1.13/commit/464719c02a503b9daaa52be255e442848adfd8c5

```
[14:41:29] [Craft Scheduler Thread - 33/INFO]: WorldEdit: player  (in "world"): /stack 5 - Orientation: WEST - Region: (768, 95, 714) - (898, 145, 814)
[14:41:30] [Craft Scheduler Thread - 33/ERROR]: [WorldEdit] An unexpected error occurred while handling a FAWE command
java.lang.NullPointerException: null
	at com.sk89q.worldedit.function.operation.ForwardExtentCopy.lambda$resume$2(ForwardExtentCopy.java:359) ~[?:?]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176) ~[?:?]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?]
	at com.sk89q.worldedit.function.operation.ForwardExtentCopy.resume(ForwardExtentCopy.java:360) ~[?:?]
	at com.sk89q.worldedit.function.operation.Operations.completeBlindly(Operations.java:68) ~[?:?]
	at com.sk89q.worldedit.EditSession.stackCuboidRegion(EditSession.java:1930) ~[?:?]
	at com.sk89q.worldedit.command.RegionCommands.lambda$stack$11(RegionCommands.java:622) ~[?:?]
	at com.boydti.fawe.object.FawePlayer.checkConfirmationStack(FawePlayer.java:223) ~[?:?]
	at com.sk89q.worldedit.command.RegionCommands.stack(RegionCommands.java:618) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at com.sk89q.worldedit.util.command.parametric.ParametricCallable.call(ParametricCallable.java:265) ~[?:?]
	at com.sk89q.worldedit.util.command.SimpleDispatcher.call(SimpleDispatcher.java:133) ~[?:?]
	at com.sk89q.worldedit.extension.platform.CommandManager$2.get(CommandManager.java:414) ~[?:?]
	at com.sk89q.worldedit.extension.platform.CommandManager.handleCommandTask(CommandManager.java:437) ~[?:?]
	at com.sk89q.worldedit.extension.platform.CommandManager.handleCommandOnCurrentThread(CommandManager.java:418) ~[?:?]
	at com.sk89q.worldedit.extension.platform.CommandManager$3$1.run(CommandManager.java:536) ~[?:?]
	at com.boydti.fawe.object.FawePlayer.lambda$runAction$3(FawePlayer.java:331) ~[?:?]
	at com.boydti.fawe.object.task.SimpleAsyncNotifyQueue.operate(SimpleAsyncNotifyQueue.java:25) ~[?:?]
	at com.boydti.fawe.object.task.AsyncNotifyQueue$1.run(AsyncNotifyQueue.java:15) ~[?:?]
	at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.13.2.jar:git-Paper-588]
	at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52) ~[patched_1.13.2.jar:git-Paper-588]
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.13.2.jar:git-Paper-588]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by MattBDev** [comment]
I'm close to narrowing the exact issue down.

**Comment by MattBDev** [comment]
Fixed in 712479a
